Dockerfile-solr-jts

```diff
@@ -1,7 +1,6 @@
-
 FROM solr:7.3
 
 MAINTAINER Forest Gregg <fgregg@datamade.us>
 
 RUN set -x \
-&& curl -o /opt/solr/server/solr-webapp/webapp/WEB-INF/lib/jts-core-1.15.0.jar https://repo1.maven.org/maven2/org/locationtech/jts/jts-core/1.15.0/jts-core-1.15.0.jar
\ No newline at end of file
+&& curl -o /opt/solr/server/solr-webapp/webapp/WEB-INF/lib/jts-core-1.15.0.jar https://repo1.maven.org/maven2/org/locationtech/jts/jts-core/1.15.0/jts-core-1.15.0.jar
```

Comment:
why did you keep this file? did you figure out what is was for?

Reply:
No idea what it's for, but it doesn't seem to have a role in deployment, so I think it's probably safe to strike. I'll give it a whirl.

Reply:
Oh, figured it out. It's because [we want to support spatial fields](https://stackoverflow.com/questions/29675983/solr-core-is-not-loaded-when-i-add-spatialcontextfactory-attribute-in-the-fi). I'm betting that we manually installed the adapter on the server.

Reply:
Relevant docs: https://lucene.apache.org/solr/guide/6_6/spatial-search.html#SpatialSearch-JTSandPolygons

Reply:
I added a comment to the Dockerfile so we know why we're using it.